sdk/monitor/Azure.Monitor.OpenTelemetry.AspNetCore/src/AzureMonitorAspNetCoreEventSource.cs

```diff
@@ -51,6 +51,15 @@ public void ConfigureFailed(Exception ex)
             }
         }
 
+        [NonEvent]
+        public void GetEnvironmentVariableFailed(string envVarName, Exception ex)
+        {
+            if (IsEnabled(EventLevel.Error))
+            {
+                GetEnvironmentVariableFailed(envVarName, ex.FlattenException().ToInvariantString());
+            }
+        }
+
```

Comment:
Please move this adjacent to your other method (just above 75). 
That makes it easier to read these.